### PR TITLE
Added KuCoin

### PR DESCRIPTION
**Pull request by ITSVision**
With pairs NEO-BTC and GAS-BTC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,7 @@
 - [HitBTC](https://hitbtc.com/NEO-to-ETH) - A European exchange that allows trading for BTC, ETH and USDT.
 - [JUBI](https://www.jubi.com/coin/ans) - A Chinese exchange that allows trading for Chinese Yuan Renminbi (CNY). *Chinese interface*
 - [Poloniex](https://www.19800.com/trade.html?marketid=cny_gas) - A United States-based exchange that allows trading for BTC and ETH (only GAS).
+- KuCoin [GAS](https://www.kucoin.com/#/trade/GAS-BTC) [NEO](https://www.kucoin.com/#/trade/NEO-BTC) - Korean Exchange doing GAS-BTC and NEO-BTC.
```
Comment (revett) [inline README.md]: 1. Please add a single link, following the format of all other links.
1. Please place the link in A-Z alphabetical order.

**Comment by ITSVision** [comment]
Sure want to have it like this?
- [KuCoin GAS](https://www.kucoin.com/#/trade/GAS-BTC) - Korean Exchange doing GAS-BTC.
- [KuCoin NEO](https://www.kucoin.com/#/trade/NEO-BTC) - Korean Exchange doing NEO-BTC.

**Comment by revett** [comment]
@ITSVision just one link for the exchange will be enough - thanks!

**Comment by ITSVision** [comment]
done

**Comment by revett** [comment]
@ITSVision thanks for the PR! 🚀

**Pull request merged**